app/src/main/java/com/example/dine_and_donate/HomeFragments/MapFragment.java

```diff
@@ -223,19 +223,13 @@ public void onCameraIdle() {
                 if (cameraLatitude == null || cameraLongitude == null) {
                     cameraLatitude = newLatitude;
                     cameraLongitude = newLongitude;
+                    generateMarkers();
                 }
 
-                if (((Math.abs(newLongitude - cameraLongitude) > 0.03)
-                        || (Math.abs(newLatitude - cameraLatitude) > 0.03))
-                        && (map.getCameraPosition().zoom > 10)) {
+                if (map.getCameraPosition().zoom > 10) {
```

Comment:
i don't totally follow what's happening here, @ryan-beck does this seem right? i know you mentioned something about weird zoom yesterday